tf2_geometry_msgs/CMakeLists.txt

```diff
@@ -46,5 +46,5 @@ install(DIRECTORY include/${PROJECT_NAME}/
 #
 #
 # endif()
-ament_export_include_directories(include)
-ament_package()
+ament_auto_find_build_dependencies()
+ament_auto_package()
```

Comment:
If you use `ament_auto_package()` the `install` directive in line 25-27 shouldn't be necessary (see https://github.com/ament/ament_cmake/blob/c65dd2dfae39c8772580a3f40a46044fda34c98f/ament_cmake_auto/cmake/ament_auto_package.cmake#L16-L26).